mlrun/__init__.py

```diff
@@ -159,6 +159,7 @@ def set_environment(
                 "artifact_path must refer to an absolute path" " or a valid url"
             )
         mlconf.artifact_path = artifact_path
+    mlconf.reload()
```

Comment:
This will override changes set directly on mlconf.
You need to:
1. Set env vars
2. Reload config
3. Configure mlconf

So move all env changes to the top and mlconf changes to the bottom

Reply:
check again